### PR TITLE
feat: add exodus wallet support

### PR DESCRIPTION
**Pull request by mzndako**
## Summary
Hi team, this PR adds Exodus Wallet Browser Extension support

### CODE REVIEW

File: wallets/exodus-extension/src/extension/registry.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Wallet } from '@cosmos-kit/core';
+import { GoDesktopDownload } from 'react-icons/go';
+import { RiChromeFill } from 'react-icons/ri';
+
+import { EXODUS_ICON } from '../constants';
+
+export const exodusExtensionInfo: Wallet = {
+  name: 'exodus-extension',
+  logo: EXODUS_ICON,
```
Comment (pyramation) [inline wallets/exodus-extension/src/extension/registry.ts:9]: @liujun93 I wonder if we should standarize this, maybe we can make a build script. This is actually more efficient to use the base64 encoding, something to consider — I like that it's both in the directory of images, but also been base64 encoded. Pretty cool feature

**Comment by pyramation** [comment]
awesome work @mzndako!

LGTM :) 

cc @twhy @liujun93

File: wallets/exodus-extension/src/extension/registry.ts (hunk 1 above)
Comment (liujun93) [reply wallets/exodus-extension/src/extension/registry.ts:9]: I used to use hex, but I can switch to base64 as you suggested. But FYI any encoding settings in wallet registry depends on the wallet not us

Comment (pyramation) [reply wallets/exodus-extension/src/extension/registry.ts:9]: totally! good catch June! I meant that it's nice that they used the encoding style. Very fast for load times. Perhaps we make a build step for this in the `wallet` folder

**Pull request merged**